webapp/components/wpt-results.html

```diff
@@ -370,9 +392,57 @@ <h3>
         return url;
       }
 
+      computeAutocompleteURL(testRuns, search) {
+        let url = '/api/autocomplete?limit=10&';
+        if (testRuns) {
+          url += `run_ids=${
+            window.encodeURIComponent(
+              testRuns.map(r => r.id.toString()).join(','))}&`;
+        }
+        if (search) {
+          url += `q=${window.encodeURIComponent(search)}`;
+        }
+        return url;
+      }
+
+      observeLoadingCount(newValue, oldValue) {
+        if (newValue === oldValue) {
+          return;
+        }
+        const elem = this.shadowRoot.querySelector('paper-spinner-lite');
+        if (!elem) {
+          return;
+        }
+
+        if (newValue && !oldValue) {
+          elem.setAttribute('active', 'true');
+        } else if (!newValue && oldValue) {
+          elem.removeAttribute('active');
+        }
+      }
+
+      constructor() {
+        super();
+        this.onSearchCommit = this.handleSearchCommit.bind(this);
+      }
+
+      connectedCallback() {
+        super.connectedCallback();
+        this.shadowRoot.querySelector('test-search')
+          .addEventListener('commit', this.onSearchCommit);
+      }
+
+      disconnectedCallback() {
+        this.shadowRoot.querySelector('test-search')
+          .removeEventListener('commit', this.onSearchCommit);
+        super.disconnectedCallback();
+      }
+
       ready() {
         super.ready();
         this.loadRuns().then(async runs => {
+          this.fetchResults();
```

Comment:
Was this intended?

Reply:
Yes. We don't get an initial fetch without this.